### PR TITLE
docs: fix clusterversion patch command

### LINKED ISSUE #151: Contributing guide issue
Hi,
following command returns an error:
https://github.com/openshift/builder/blame/master/CONTRIBUTING.md#L34

```
$ oc patch clusterversion/version --patch '{"spec":{"overrides":[{"kind":"ConfigMap", "namespace":"openshift-controller-manager-operator","name":"openshift-controller-manager-images","unmanaged":true}]}}' --type=merge
The ClusterVersion "version" is invalid: spec.overrides.group: Required value
```

### PR DESCRIPTION
**Pull request by danielkucera**
Fixes #151

### CODE REVIEW

**Comment by gabemontero** [comment]
/ok-to-test
/lgtm

thanks @danielkucera !!

/assign @adambkaplan 
for approval

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ After your cluster is installed, you will need to do the following:
 1. Patch the cluster version so that you can launch your own builder image:
 
 ```
-$ oc patch clusterversion/version --patch '{"spec":{"overrides":[{"kind":"ConfigMap", "namespace":"openshift-controller-manager-operator","name":"openshift-controller-manager-images","unmanaged":true}]}}' --type=merge
+$ oc patch clusterversion/version --patch '{"spec":{"overrides":[{"kind":"ConfigMap", "namespace":"openshift-controller-manager-operator","name":"openshift-controller-manager-images","unmanaged":true, "group": "apps/v1"}]}}' --type=merge
```
Comment (adambkaplan) [inline CONTRIBUTING.md:34]: Why does `apps/v1` work as a group? `ConfigMap` is a part of the `core` api group, not `apps`.

Comment (gabemontero) [reply CONTRIBUTING.md:34]: the CVO's interpretation of these fields are not as literaly as one might think @adambkaplan 

But at least for me I can confirm from past usage that for core api objs like `ConfigMaps` or `Deployments`, when I override CVO managed items of those types, I have had to put `apps/v1` in there.

Full transparency though, I have not experimented with it recently to see if it takes say just `apps`, but I can say as recently as yesterday `apps/v1` worked for me as well as what @danielkucera saw.

**Comment by rhatdan** [comment]
@danielkucera @adambkaplan What is going on with this PR?  Is it still valid?

**Comment by adambkaplan** [comment]
@rhatdan I may want to close this and open a new PR. Patching the CVO with overrides is clunky - for developers hacking on OpenShift the easier thing to do is to scale down the CVO to 0 replicas:

```
$ oc scale deployment cluster-version-operator -n openshift-cluster-version --replicas=0
```

**Pull request closed without merging**